helm/docs/configuring-delegated-operators.md

```diff
@@ -25,13 +25,13 @@
     - [List Merges](#list-merges)
 - [Migrating from `delegatedOperatorExecutorSettings` to `delegatedOperatorDeployments`](#migrating-from-delegatedoperatorexecutorsettings-to-delegatedoperatordeployments)
   - [Example](#example)
-- [Prior to v2.7.0](#prior-to-v270)
+- [Prior to v2.8.0](#prior-to-v280)
 
 <!-- tocstop -->
 
-## v2.7.0+
+## v2.8.0+
 
-As of version 2.7.0, `delegatedOperatorExecutorSettings`
+As of version 2.8.0, `delegatedOperatorExecutorSettings`
```

Comment:
I don't think you want to bump this?  Seems like it was bumped in 2.7.0 and we should leave it like that?